### PR TITLE
i18(zh-cn): Synchronizing content

### PR DESCRIPTION
**Pull request by GuoJikun**
#### Description

- update the `zh-cn/plugin`  directory(related #2298 )
- update `zh-cn/rss.mdx`(related #2314 )

### CODE REVIEW

File: src/content/docs/zh-cn/rss.mdx
Comment (dklassic) [inline src/content/docs/zh-cn/rss.mdx:1]: A bit weird that the rendered site doesn't seem to show the description properly, but I see no syntax error or stuffs.

**Review by dklassic — APPROVED** [review]

**Comment by dklassic** [comment]
@GuoJikun do you mind fixing the formatting?

https://github.com/tauri-apps/tauri-docs/actions/runs/9854367981/job/27213449316?pr=2352

File: src/content/docs/zh-cn/rss.mdx
Comment (GuoJikun) [reply src/content/docs/zh-cn/rss.mdx:1]: The display is normal at `https://deploy-preview-2352--tauri-v2.netlify.app`.

**Comment by GuoJikun** [comment]
> @GuoJikun do you mind fixing the formatting?
> 
> https://github.com/tauri-apps/tauri-docs/actions/runs/9854367981/job/27213449316?pr=2352

Don't mind.

**Comment by GuoJikun** [comment]
@dklassic all done.

**Pull request merged**